src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Overview/EditDraftReturnedNotesOverview.cshtml

```diff
@@ -20,7 +20,7 @@
 
                     @using (Html.BeginForm("Index", "ManageEvidenceNotes", FormMethod.Get))
                     {
-                        @Html.Hidden("tab", Model.ActiveOverviewDisplayOption)
```

Comment:
@GuidoWorkDev 
I don't think this will work. The reason for the hidden input has this name is that is gets mapped to the action parameters i.e. "tab" in the index action.

Index(Guid organisationId, Guid aatfId, 
            string tab = null,
            ManageEvidenceNoteViewModel manageEvidenceNoteViewModel = null)


I think we may have to change the 
@using (Html.BeginForm("Index", "ManageEvidenceNotes", FormMethod.Get))

to something like

@using (Html.BeginForm("Index", new RouteValueDictionary() { { "tab", Model.ActiveOverviewDisplayOption } }, "ManageEvidenceNotes", FormMethod.Get))

to remember the selected tab.

Reply:
done

Reply:
ta... will merge in a bit later